styles.css

```diff
@@ -3,5 +3,197 @@
 }
 
 body {
-    background-color: rgb(234, 234, 234);
+    background-color: white;
+    height: 100vh;
+}
+
+#blue-box{
+
+    display: flex;
+    justify-content: center;
+    padding-top: 50px;
+    
+}
+
+#telegram{
+    height: 30px;
+    width: 30px;
+}
+
+#join-us{
+    height: 40vh;
```

Comment:
no need to specify any height here. Since if you size the children elements properly and also the padding and margins for the child elements it will suffice. 

Also if you suffice, then there will be a overflow in y direction for this element in smaller screens. So removing this is the best option.